### PR TITLE
feat(bridge-ui): custom recipient address

### PR DESCRIPTION
**Pull request by cyberhorsey**

### CODE REVIEW

File: packages/bridge-ui/src/App.svelte [hunk 1]
```diff
@@ -146,31 +146,44 @@
...
       const apiTxs = await $relayerApi.GetAllByAddress(userAddress);
 
+      console.log(apiTxs);
+
       const blockInfoMap = await $relayerApi.GetBlockInfo();
       relayerBlockInfoMap.set(blockInfoMap);
 
       const txs = await $transactioner.GetAllByAddress(userAddress);
 
+      console.log(txs);
```
Comment (d1onys1us) [inline packages/bridge-ui/src/App.svelte:156]: do we want to keep these console.log's? i don't mind, but sometimes it's a lint rule `no-console-log`. same with below.

File: packages/bridge-ui/src/components/form/BridgeForm.svelte [hunk 2]
```diff
@@ -222,6 +224,11 @@
     try {
       loading = true;
       if (requiresAllowance) throw Error('requires additional allowance');
+      if (showTo) {
+        if (!ethers.utils.isAddress(to)) {
```
Comment (d1onys1us) [inline packages/bridge-ui/src/components/form/BridgeForm.svelte:228]: can combine this into single if block

File: packages/bridge-ui/src/components/form/To.svelte [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+  import ButtonWithTooltip from '../ButtonWithTooltip.svelte';
+  import TooltipModal from '../modals/TooltipModal.svelte';
+
+  export let showTo: boolean = false;
```
Comment (d1onys1us) [inline packages/bridge-ui/src/components/form/To.svelte:5]: do these need to be exported? i don't see them as imported in BridgeForm

Comment (cyberhorsey) [reply packages/bridge-ui/src/components/form/To.svelte:5]: theyre not imported in BridgeForm, theyre bound to using `bind` value to make the values in thsi component available in the parent BridgeForm component.

Comment (cyberhorsey) [reply packages/bridge-ui/src/components/form/To.svelte:5]: theyre not imported in BridgeForm, theyre bound to using `bind` value to make the values in thsi component available in the parent BridgeForm component.

File: packages/bridge-ui/src/App.svelte (hunk 1 above)
Comment (cyberhorsey) [reply packages/bridge-ui/src/App.svelte:156]: will remove!

File: packages/bridge-ui/src/components/form/BridgeForm.svelte (hunk 2 above)
Comment (cyberhorsey) [reply packages/bridge-ui/src/components/form/BridgeForm.svelte:228]: will do!

**Review by dionysuzx — DISMISSED** [review]

File: packages/bridge-ui/src/App.svelte [hunk 4]
```diff
@@ -146,6 +146,8 @@
 
       const apiTxs = await $relayerApi.GetAllByAddress(userAddress);
 
+      console.log(apiTxs);
```
Comment (shadab-taiko) [inline packages/bridge-ui/src/App.svelte:149]: Console.log
```suggestion

```

**Review by shadab-taiko — DISMISSED** [review]

**Review by davidtaikocha — APPROVED** [review]

**Review by dionysuzx — APPROVED** [review]

**Pull request merged**